modules/navbar.js

```diff
@@ -0,0 +1,27 @@
+// const listNav = document.getElementById('list-page');
+// const addNewNav = document.getElementById('add-list');
+// const contactNav = document.getElementById('contact-list');
```

Comment:
- kindly Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.👍